lms/envs/common.py

```diff
@@ -3923,3 +3923,9 @@ def _make_locale_paths(settings):  # pylint: disable=missing-function-docstring
 
 ##################### SUPPORT URL ############################
 SUPPORT_HOW_TO_UNENROLL_LINK = ''
+
+######################## Setting for django-countries ########################
+# django-countries provides an option to make the desired countries come up in
+# selection forms, if left empty countries will come up in ascending order as before
+# https://github.com/SmileyChris/django-countries#show-certain-countries-first
+COUNTRIES_FIRST = []
```

Comment:
@farhaanbukhsh How is this going to be override-able from the `.env` files? You missed an entry in `production.py` that reads from the *yaml* files.